README.md

```diff
@@ -17,30 +18,32 @@ domains:
 
 
 ## Building
+ 
+To build
 
-To build:
-
-    docker build . -t ghcr.io/opensafely-core/opensafely-proxy
+    make build
```

Comment:
default for IMAGE_NAME is `proxy` which seems like a substantive change to this command?

Reply:
Yes, the old command was out of date, now we have automatic publishing. It's cleaner to have the default dev env build use a local tag like `proxy`, rather the then publication tag of `ghcr.io/opensafely-core/proxy`. Developers can manually tag it as such locally if they need to. But publishing is done via GH action now, rather than manually, so it's clear to a) reuse the Makefile build step and b) only tag it with a local name